tests/rkt_net_test.go

```diff
@@ -381,7 +381,7 @@ func TestNetTemplates(t *testing.T) {
 		Type: "ptp",
 		Ipam: ipamTemplateT{
 			Type:   "host-local",
-			Subnet: "10.1.3.0/24",
+			Subnet: "11.11.3.0/24",
```

Comment:
Can you add a comment somewhere with a link to the wikipedia page about the `11.*` IP range?